assets/src/scss/pages/_author.scss

```diff
@@ -1,29 +1,68 @@
 .author {
+  .page-header {
+    display: flex;
+    justify-content: flex-start;
+    gap: 40px;
```

Comment:
Not a big deal, but you could use one of our spacing variables here, aka `$sp-5`

Reply:
I'm a bit hesitant about using those, as they make our code significantly less accessible. With using a string value of `40px`, anyone can look at a single file alone and understand what it does.

Whereas using a variable to hide these values behind a multiplication makes it impossible to understand this file without looking up the variable definition.

Reply:
...